indigo/distribution.yaml

```diff
@@ -6172,6 +6172,21 @@ repositories:
       url: https://github.com/fkie/multimaster_fkie.git
       version: indigo-devel
     status: maintained
+  multirobot_tracker:
+    doc:
+      type: git
+      url: https://github.com/multirobot-ferr/multirobot_tracker.git
+      version: indigo-devel
```

Comment:
This must be a valid branch name in the repo. Likely `master` in your case.

Same for the `source` entry below.